src/itty-router.js

```diff
@@ -1,30 +1,32 @@
-const Router = (options = {}) =>
-  new Proxy(options, {
-    get: (obj, prop, receiver) => prop === 'handle'
-      ? async (request, ...args) => {
-          for (let [route, handlers] of obj.routes.filter(i => i[2] === request.method || i[2] === 'ALL')) {
-            let match, response, url
-            if (match = (url = new URL(request.url)).pathname.match(route)) {
-              request.params = match.groups
-              request.query = request.query || Object.fromEntries(url.searchParams.entries())
-
-              for (let handler of handlers) {
-                if ((response = await handler(request.proxy || request, ...args)) !== undefined) return response
-              }
-            }
-          }
+const Router = ({ base = '' } = {}, routes = []) => ({
```

Comment:
That costs a byte, you sure? It's still accessible introspection-wise as `Router({}, routes)`.

Reply:
Torn on this to be honest... if we leave it under options, it actually gets renamed to simply `r` in the production minified version (rather than `routes`).  That's a bit awkward to describe in documentation, but then so is a second param for introspection/preloading, especially if you didn't have other params:

```js
const r = [] // routes

const router = Router({}, r) // awkward
const router = Router(null, r) // awkward
const router = Router({ r }) // awkward, but less-so?
const router = Router({ routes }) // nice, but then we lose a chunk of chars...
```